File: java/src/main/java/com/mertsy/javasample/capturing/CarsAndVehiclesActivity.java
package com.mertsy.javasample.capturing;

import android.os.Bundle;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.mertsy.capturing.MertsyCapturing;
import com.mertsy.javasample.ClipboardUtil;
import com.mertsy.javasample.R;
import com.mertsy.capturing.CapturingResultListener;
import com.mertsy.capturing.MertsyCapturingView;

public class CarsAndVehiclesActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_cars_and_vehicles);

        MertsyCapturingView capturingView = findViewById(R.id.carsAndVehicles);

        capturingView.setBackPressedDispatcher(getOnBackPressedDispatcher());
        capturingView.setFragmentManager(getSupportFragmentManager());
        capturingView.setLifecycleOwner(this);
        capturingView.setCapturingResultListener(new CapturingResultListener() {
            @Override
            public void onComplete(@NonNull String modelId) {
                new ClipboardUtil(getApplicationContext()).copy(modelId);
                Toast.makeText(getApplicationContext(), "Copied to clipboard\n" +  modelId , Toast.LENGTH_LONG).show();                finish();
            }

            @Override
            public void onCancel() {
                finish();
            }
        });
    }

}

File: kotlin/src/main/java/com/mertsy/kotlinsample/capturing/CarsAndVehiclesActivity.kt
package com.mertsy.kotlinsample.capturing

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.mertsy.capturing.CapturingResultListener
import com.mertsy.capturing.MertsyCapturingView
import com.mertsy.kotlinsample.ClipboardUtil
import com.mertsy.kotlinsample.R

class CarsAndVehiclesActivity : AppCompatActivity(R.layout.activity_cars_and_vehicles) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        with(findViewById<MertsyCapturingView>(R.id.carsAndVehicles)) {
            setBackPressedDispatcher(onBackPressedDispatcher)
            setFragmentManager(supportFragmentManager)
            setLifecycleOwner(this@CarsAndVehiclesActivity)
            setCapturingResultListener(object : CapturingResultListener {
                override fun onComplete(modelId: String) {
                    ClipboardUtil(applicationContext).copy(modelId)
                    Toast.makeText(
                        applicationContext,
                        "Copied to clipboard\n$modelId",
                        Toast.LENGTH_LONG
                    ).show()
                    finish()
                }

                override fun onCancel() {
                    finish()
                }
            })
            /*
            *
            * OR
            *
            * setCapturingResultCallback(onComplete = {}, onCancel = {})
            *
            * */
        }

    }

}